integrations/visual_studio/creating_packages.rst

```diff
@@ -0,0 +1,228 @@
+
+This process can be repeated to create and test packages for different configurations:
+
+.. code-block:: bash
+
+   $ conan create . demo/testing -s arch=x86
+   $ conan create . demo/testing -s compiler="Visual Studio" -s compiler.runtime=MDd -s build_type=Debug
+   $ conan create . demo/testing -s compiler="Visual Studio" -s compiler.runtime=MDd -s build_type=Debug -s arch=x86
+
+.. note::
+
+    It is not mandatory to specify the ``compiler.runtime`` setting. If it is not explicitly
+    defined, Conan will automatically use ``runtime=MDd`` for ``build_type==Debug`` and
+    ``runtime=MD`` for ``build_type==Release``.
+
+
+You can list the different created binary packages:
+
+.. code-block:: bash
+
+    $ conan search hello/0.1@demo/testing
+
+
+Reusing packages
+----------------
+
+To use existing packages directly from Visual Studio, Conan provides the ``MSBuildDeps``
+generator. Let's clone an existing "Chat" project, consisting of a ChatLib static library that
+makes use of the previous "Hello World" package, and a MyChat application, calling the ChatLib
+library function.
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples
+    $ cd features/visual_studio/chat
+
+As above, the repository contains a Visual Studio solution in the ``build`` folder. But if
+you try to open it, it will fail to load. This is because it is expecting to find a file with
+the required information about dependencies, so it is necessary to obtain that file first. Just
+run:
+
+.. code-block:: bash
+
+    $ conan install .
+
+You will see that it created several `*.props` files, containing the Visual Studio
+properties (like ``<AdditionalIncludeDirectories>``), so it is able to find the installed
+dependencies.
+
+Now you can open the IDE and build and run the app (by the way, the chat function is just calling
+the ``hello()`` function two or three times, depending on the build type):
+
+.. code-block:: bash
+
+    $  build\ChatLib\ChatLib.sln
+    # Switch to Release
+    # MyChat -> Set as Startup Project
+    # Ctrl + F5 (Debug -> Run without debugging)
+    > Hello World Release!
+    > Hello World Release!
+
+If you wish to link with the debug version of Hello package, just install it and change IDE build
+type:
+
+.. code-block:: bash
+
+    $ conan install . -s build_type=Debug -s compiler="Visual Studio" -s compiler.runtime=MDd
+    # Switch to Debug
+    # Ctrl + F5 (Debug -> Run without debugging)
+    > Hello World Debug!
+    > Hello World Debug!
+    > Hello World Debug!
+
+Now you can close the IDE and clean the temporary files:
+
+.. code-block:: bash
+
+    # close VS IDE
+    $ git clean -xdf
+
+Again, there is a ``conanfile.py`` package recipe in the repository, together with a
+``test_package``. The recipe is almost identical to the above one, just with two minor
+differences:
+
+.. code-block:: python
+
+    requires = "hello/0.1@demo/testing"
+    ...
+    generators = "visual_studio"
```

Comment:
Wrong, legacy generator. I strongly recommend actually running this docs and the example code against the Conan 2.0-alpha

Reply:
The examples in https://github.com/conan-io/examples are updated and tested. I just forgot to update this snippet. Fixed.